week-3/reading-list/script.js

```diff
@@ -21,3 +21,23 @@ const books = [
   },
 ];
 
+const ul = document.querySelector('#reading-list');
+for (const book of books) {
+  const li = document.createElement("li");
+  ul.appendChild(li);
+  //Display title of the book
+  const bookName = document.createElement("p");
+  bookName.innerText = book.title + "-------"+book.author;
+  li.appendChild(bookName);
+ //Display author of the book
```

Comment:
Your comments seem a little unrelated to the code they're each applying to - I'd maybe remove them.